src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -334,6 +334,7 @@ public PublicClientApplicationBuilder WithKerberosTicketClaim(string servicePrin
         }
 
         /// <summary>
+        /// This method is being deprecated
         /// Returns <c>true</c> if a broker can be used.

```

Comment:
I think Obsolete attribute is enough. Plus "being deprecated" implies an ongoing process, while this is already obsolete.
```suggestion
        /// Returns <c>true</c> if a broker can be used.

```

Reply:
Good point. I incorporated Travis's wording. Changed to "has been".